aspnetcore/performance/diagnostic-tools.md

```diff
@@ -11,21 +11,25 @@ uid: performance/diagnostic-tools
 
 By [Mike Rousos](https://github.com/mjrousos)
 
-This topic lists tools for diagnosing performance issues in ASP.NET Core.
+This article lists tools for diagnosing performance issues in ASP.NET Core.
 
 ## Visual Studio Diagnostic Tools
 
-The [profiling and diagnostic tools](https://docs.microsoft.com/visualstudio/profiling) built into Visual Studio 2017 are a great place to start investigating performance issues since they're powerful and easy to use from the Visual Studio development environment. The tooling allows analysis of CPU usage, memory usage, and performance events in ASP.NET Core applications.
+The [profiling and diagnostic tools](/visualstudio/profiling) built into Visual Studio are a good place to start investigating performance issues. These tools are powerful and convenient to use from the Visual Studio development environment. The tooling allows analysis of CPU usage, memory usage, and performance events in ASP.NET Core apps.
 
-More information is available in [Visual Studio documentation](https://docs.microsoft.com/visualstudio/profiling/profiling-overview).
+More information is available in [Visual Studio documentation](/visualstudio/profiling/profiling-overview).
 
 ## Application Insights
 
-[Application Insights](https://docs.microsoft.com/azure/application-insights/app-insights-overview) can provide in-depth performance data for your application. Application Insights automatically collects data on response rates, failure rates, dependency response times, and more. You can also log custom events and metrics specific to your app. 
+[Application Insights](/azure/application-insights/app-insights-overview) provides in-depth performance data for your app. Application Insight automatically collects data on response rates, failure rates, dependency response times, and more. Application Insights supports logging custom events and metrics specific to your app.
```

Comment:
Regarding the change to the second sentence here, I think it should remain Application Insights. The name of the service is always plural in my experience.